app/views/internal/response_templates/edit.html.erb

```diff
@@ -0,0 +1,3 @@
+<h1>Editing <%= @response_template.title %></h1>
+<br>
```

Comment:
Can we remove this? The h2 should already have some margin, but we can add more with bootstrap's margin utility class if we need to!

Resource for those classes: https://getbootstrap.com/docs/4.4/utilities/spacing/